### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by icandrew**
@cheyenneIT22

### CODE REVIEW

**Comment by lindsaymarkward** [comment]
Please revisit the teaching on descriptive commit messages in the imperative mood.
When you mention someone for a code review, include a short request with some detail of what you want done.

File: prac_05/state_names.py [hunk 1]
```diff
@@ -5,14 +5,14 @@
...
 
 # TODO: Reformat this file so the dictionary code follows PEP 8 convention
-CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT" : "Northern Territory", "WA" : "Western Australia",
-            "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
 print(CODE_TO_NAME)
 
-state_code = input("Enter short state: ")
+state_code = input("Enter short state: ").upper()
```
Comment (cheyenneIT22) [inline prac_05/state_names.py:12]: is this meant to be only one input with the .upper()?

File: prac_05/hex_colours.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 while color_name != "":
     colors = (color.lower() for color in COLOR_TO_HEX.keys())
     if color_name in colors:
-        print(f"{color_name} is available")
```
Comment (cheyenneIT22) [inline prac_05/hex_colours.py]: is the is available suppose to have the underscore?

**Review by cheyenneIT22 — COMMENTED** [review]
Great code!! I apologize for taking so long to leave a review.

**Pull request merged**